src/memcached-store.ts

```diff
@@ -1,49 +0,0 @@
-import * as Memcached from 'memcached'
-
-export interface IMemcachedStoreOptions {
-  expiration?: number,
-  prefix?: string,
-  client?: any,
-}
-
-export default class MemcachedStore {
-  public prefix: string
-  public expiration: number
```

Comment:
Maybe we should keep this and add an `incr()` method to the new store for backwards-compatibility with older exress-rate-limit and current express-slow-down?

Also, then we could check in `init()` and verify that expiration matches windowMs/1000 and throw an error if they disagree.